sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/SealedEnumProcessor.kt

```diff
@@ -85,19 +100,36 @@ internal class SealedEnumProcessor(
          */
         val sealedEnumAnnotations = sealedClassElement
             .getAnnotationsByType(GenSealedEnum::class.java)
-            .map { it.traversalOrder }
-            .distinct()
+
+        // Ensure that the annotation are unique by traversal order
+        if (!sealedEnumAnnotations.areUniqueBy { it.traversalOrder }) {
+            printError(ERROR_ELEMENT_IS_ANNOTATED_WITH_REPEATED_TRAVERSAL_ORDER)
+            return null
+        }
 
         /**
          * If the list of [GenSealedEnum] annotations is empty, then we must be trying to generate [SealedEnum] from
          * all sealed classes.
          */
```

Comment:
This is specifically for handling the `sealedenum.autoGenerateSealedEnums` processor option, which will process _all_ `@Metadata` annotations (in other words, all Kotlin classes) and generate `SealedEnum`s for all `sealed classes` that match the criteria of having only objects. In that case, we use a default setting since there might not be a `GenSealedEnum` annotation on a class that we're processing.

Reply:
This code was removed with the removal of the autogeneration feature.